client/my-sites/plans-features-main/index.jsx

```diff
@@ -247,7 +247,7 @@ export class PlansFeaturesMain extends Component {
 					selected={ customerType === 'business' }
 					path={ '?customerType=business' }
 				>
-					{ translate( 'Online Stores and Business Sites' ) }
+					{ translate( 'Business Sites and Online Stores' ) }
```

Comment:
:disappointed: 20 existing translations will be lost with this change.
If the string must change, the following option has already been translated **20** times:
`translate( 'Stores and Business Sites' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).